index.js

```diff
@@ -1,12 +0,0 @@
-"use strict"
-
-const {loadNativeCerts} = require("./binding.node")
-module.exports = () => loadNativeCerts().map(format)
-
-function format(buf) {
-    return (
-        "-----BEGIN CERTIFICATE-----\n" +
-        buf.toString("base64").match(/.{1,72}/g).join("\n") +
-        "\n-----END CERTIFICATE-----"
-    )
-}
```

Comment:
I'm not sure how this would work together with the index.js that `napi-rs` generates.

In the examples they show doing everything in Rust, and then the `#[napi]` functions are exposed as JS functions, which appear in the generated index.js. Would this not overwrite any pre-existing index.js?

Reply:
There's nothing of real interest in the generated index.js, is there? In other projects I just run the project generator once to set up the skeleton, then toss out everything I don't need.